### ISSUE TITLE
cat list-of-list.parquet

### ISSUE THREAD

**Issue by hangxie** [hangxie/parquet-tools#211]
Something's wrong:

```
$ parquet-tools version
v1.19.0
$ parquet-tools cat testdata/list-of-list.parquet
[]
$ parquet-tools row-count testdata/list-of-list.parquet
5
```

**Comment by hangxie** [comment]
Turned out LZO and BROTLI are not supported https://github.com/xitongsys/parquet-go/tree/master#compression-type

**Comment by hangxie** [comment]
Fixed by #212 and #213
